google_guest_agent/system_windows.go

```diff
@@ -72,3 +105,108 @@ func deleteRegKey(key, name string) error {
 
 	return k.DeleteValue(name)
 }
+
+func windowsStartService(servicename string) error {
```

Comment:
for this set of functions starting with windows:

1. how has testing gone? is using the `runCmd` function sufficient, which relies on checking exit code is 0? i don't know how this works on windows
2. do i understand the distinction between these functions is we use 'net' to start or stop a service, and we use 'sc' to disable or enable auto-start-on-boot ?
3. i think the naming could be clearer. we mix windowsActionService and windowsServiceAction naming, and the enable/disable functions both say 'start' in the name, which makes a little difficult to read at a glance. happy to offer alternative suggestions

Reply:
1. Yeah, this seems to work as expected in practice.
2. Yes, the first functions are starting/stopping the service, the others are setting the "start-up type" for the service to control what happens when the system is rebooted.
3. I updated all of the names. I think they are clearer, but open to suggestions.